ha_cluster_exporter.go

```diff
@@ -73,15 +73,33 @@ func landingpage(w http.ResponseWriter, r *http.Request) {
 		</html>`))
 }
 
+func loglevel(opt string) {
+	switch opt {
+	case "error":
+		log.SetLevel(log.ErrorLevel)
+	case "warn":
+		log.SetLevel(log.WarnLevel)
+	case "info":
+		log.SetLevel(log.InfoLevel)
+	case "debug":
+		log.SetLevel(log.DebugLevel)
+	}
+}
+
 var (
 	clock      Clock = &SystemClock{}
 	portNumber       = flag.String("port", "9002", "The port number to listen on for HTTP requests.")
+	ip4              = flag.String("ip4", "0.0.0.0", "The IP to listen on for HTTP requests.")
+	logLevel         = flag.String("level", "info", "The level of logs to log")
 )
 
 func main() {
 	// read cli option and setup initial stat
 	flag.Parse()
 
+	// Log Level
```

Comment:
this comment is redundant, I can read it from the function that we are setting the logLevel :grin: 

either you remove it, or you add more context.

Like 

```setup loglevel accordingly from user CLI flag , default is info. ```

But again, Perhaps we should just remove it.

Reply:
Yes, I did intend to add some more context- forgot to do this :laughing: